From 4213e59bed0b6cac7e1fdbc6283dc2f2456a424e Mon Sep 17 00:00:00 2001
From: Maxwell G <maxwell@gtmx.me>
Date: Tue, 10 Jun 2025 14:56:31 -0500
Subject: [PATCH 2/2] Replace pkginfo2 with pkginfo

pkginfo2 is a not-very-well-maintained fork of pkginfo and breaks with
the latest metadata 2.4 standard.
At least until https://github.com/aboutcode-org/pkginfo2/issues/5 is
fixed, let's use plain pkginfo.
---
 setup.cfg                | 2 +-
 src/packagedcode/pypi.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 7c45f388fd..5d7f109102 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -100,7 +100,7 @@ install_requires =
     parameter-expansion-patched >= 0.3.1
     pdfminer.six >= 20251227
     pefile >= 2020.1.1
-    pkginfo2 >= 30.0.0
+    pkginfo
     pip-requirements-parser >= 32.0.1
     pluggy >= 1.0.0
     plugincode >= 32.0.0
diff --git a/src/packagedcode/pypi.py b/src/packagedcode/pypi.py
index b5588ed7ca..34b28dcb5e 100644
--- a/src/packagedcode/pypi.py
+++ b/src/packagedcode/pypi.py
@@ -28,7 +28,7 @@ import dparse2
 import importlib_metadata
 import packvers as packaging
 import pip_requirements_parser
-import pkginfo2
+import pkginfo as pkginfo2
 from commoncode import fileutils
 from commoncode.fileutils import as_posixpath
 from commoncode.resource import Resource
-- 
2.53.0

